test/users/engin/sparse_bulk_dist/sparseAdd2D.chpl

```diff
@@ -7,7 +7,7 @@ const FullDom = {0..#N, 0..#N} dmapped Block({0..#N, 0..#N});
 
 config type layoutType = DefaultDist;
 var layout = new layoutType;
-var FullSparseDom: sparse subdomain(FullDom) dmapped new dmap(layout);
+var FullSparseDom: sparse subdomain(FullDom);
```

Comment:
These test changes look unrelated to what you described in the PR message. I don't mind if they go in the same PR but the PR message should at least say it also removes some no-longer-necessary code in tests, assuming that's what's going on.